### DISCUSSION TITLE
Reference Inventory File in Github Repo

### DISCUSSION THREAD

**Discussion by richardphi1618** [semaphoreui/semaphore#1926] [Q&A] [answered]
currently in my repo I have something resembling the following

inventories/
├── production
│   ├── group_vars
│   │   ├── Group1.yml
│   │   ├── Group2
│   │   │   ├── vars.yml
│   │   │   └── vault.yml
│   │   ├── Grouo3.yml
│   │   ├── Group4
│   │   │   ├── vars.yml
│   │   │   └── vault.yml
│   │   └── all.yml
│   ├── host_vars
│   │   ├── Host1
│   │   │   ├── vars.yml
│   │   │   └── vault.yml
│   │   ├── Host2.yml
│   │   ├── Host3
│   │   │   ├── var.yml
│   │   │   └── vault.yml
│   └── hosts
└── staging
   ├── group_vars
   │   ├── Group1.yml
   │   ├── Group2
   │   │   ├── vars.yml
   │   │   └── vault.yml
   │   ├── Grouo3.yml
   │   ├── Group4
   │   │   ├── vars.yml
   │   │   └── vault.yml
   │   └── all.yml
   ├── host_vars
   │   ├── Host1
   │   │   ├── vars.yml
   │   │   └── vault.yml
   │   ├── Host2.yml
   │   ├── Host3
   │   │   ├── var.yml
   │   │   └── vault.yml
   └── hosts
   
   but in semaphore for declaring inventories I only see options to point to a file or to write your inventory there inside semaphore. 
   
   Can I reference my inventory file saved in my github repo in semaphore similar to how I can reference playbooks? Maybe this is a pro feature? I am test driving using your selfhosted option for the moment.

**Comment by kikimora52** [comment, marked as answer]
I had a similar question and was able to to find a small blurb in the docs ( see item 4 bullet 1 on creation). I can confirm it does work.

https://docs.semui.co/user-guide/inventory

**Reply by richardphi1618** [reply]
Thank you so much! yes that does indeed work for me.
